tpi/terraform.py

```diff
@@ -18,22 +17,14 @@ class TPIException(Exception):
 
 
 class TerraformBackend:
```

Comment:
Feels like we should maybe rename this? The base/terraform backend naming is another leftover from dvc-machine and doesn't really have any meaning in TPI.